zig-mode.el

```diff
@@ -70,6 +70,17 @@
   :safe #'stringp
   :group 'zig-mode)
 
+(defcustom zig-flymake t
+  "Enable zig flymake integration."
+  :type 'boolean
+  :safe #'booleanp
+  :group 'zig-mode)
```

Comment:
`:group` is redundant in all these `defcustom`s because it defaults to the group created by the `defgroup` above.